cluster/gce/gci/configure-helper.sh

```diff
@@ -224,6 +224,11 @@ function create-master-auth {
   fi
   local -r basic_auth_csv="${auth_dir}/basic_auth.csv"
   if [[ -n "${KUBE_PASSWORD:-}" && -n "${KUBE_USER:-}" ]]; then
+    if [[ -e "${basic_auth_csv}" && "${METADATA_CLOBBERS_CONFIG:-false}" == "true" ]]; then
+      sed -i '/,admin,system:masters$/d' "${basic_auth_csv}"
+      # The following is for the legacy form of the password line.
+      sed -i '/,admin$/d' "${basic_auth_csv}"
```

Comment:
I don't think multiple sed removals for all versions of a line this script ever added is sustainable... down this path lies madness. Perhaps a step back to consider what data is user controlled and how we want to merge that with auto-managed data would be good.